### DISCUSSION TITLE
What's the use case of `{ 'bufnr' }` in `renderers.file`?

### DISCUSSION THREAD

**Discussion by nyngwang** [nvim-neo-tree/neo-tree.nvim#538] [Q&A] [answered]
As title. With or without it I don't see any difference on the resulting neo-tree: no buffer number is shown.

**Comment by cseickel** [comment, marked as answer]
It has no effect in the `filesystem` source, but it does show the buffer number in the `buffers` source. The renderers can technically be specified individually on each source, but since they are 99% the same for all sources, I chose to define them once at the global level and just let the unused components resolve to being empty for cases like that.

**Reply by nyngwang** [reply]
Thank you!
